app/views/businessmatching/updateservice/add/what_do_you_do_here.scala.html

```diff
@@ -46,35 +46,35 @@
             @if(msbServiceValues.contains("01")) {
                 @checkbox(
                     f = f("msbServices[]"),
-                    labelText = "businessmatching.updateservice.whatdoyoudohere.list.lbl.01",
+                    labelText = "businessmatching.updateservice.whatdoyoudohere.list.lbl.03",
```

Comment:
Again; is this a safe change given we have changed the label however; the value we pass back will still be stored as "01"?

Reply:
^